main/src/main/res/navigation/navigation_activity_main.xml

```diff
@@ -25,7 +25,8 @@
 
     <fragment
         android:id="@+id/item_menu_activity_main_advertise"
-        android:name="net.androidweekly.latestissue.LatestIssueFragment" />
+        android:name="net.androidweekly.latestissue.LatestIssueFragment"
+        android:label="@string/adversite" />
```

Comment:
you should follow the same naming convention for all labels `title_menu_activity_main_navigation_view_advertise`